### ISSUE TITLE
How do jupyter notebook be embeded in html ?

### ISSUE THREAD

**Issue by saliormoon** [jupyter/jupyter-js-notebook#235]
I want to use jupyter-js-notebook to write and run codes, can anyone give me any suggestions about how to do this? High appreciation.

**Comment by blink1073** [comment]
Hi @saliormoon, we have moved this development to JupyterLab.  We have an example [here](https://github.com/jupyterlab/jupyterlab/tree/master/examples/notebook).

**Comment by saliormoon** [comment]
ok, thank you very much.

**Comment by saliormoon** [comment]
Yeah, jupyter notebook can load js or css after installed. But now I have a html and I want to use jupyter notebook in this html to achieve the goal that one can write and run codes in the html.

**Comment by blink1073** [comment]
Hi @saliormoon, is your intent to use a custom version of the Jupyter Notebook in another application, or to output `html` into a standard Jupyter Notebook?  If it is the latter, you can use the `%%html` magic provided by the default `python` kernel.

**Comment by saliormoon** [comment]
Thank you.Yeah, it is a html page, and I want to embedded Jupyter Notebook in it to write and run my own codes and return the results in the html, just like  http://139.196.209.143/try/plugin.php.

**Comment by saliormoon** [comment]
Thank you, @blink1073. Yeah, I want to use a custom version of the Jupyter Notebook in another application, like html. There is a html page, and I want to embedded Jupyter Notebook in it to write and run my own codes and return the results in the html, just like http://139.196.209.143/try/plugin.php.

**Comment by blink1073** [comment]
Great, then the example linked above and the `%%html` magic should do what you want.

**Comment by saliormoon** [comment]
Ok, thank you much. And is there any example of %%html magic?

**Comment by blink1073** [comment]
Here is an example (note that it prints to the JavaScript console of the browser.

<img width="512" alt="screen shot 2016-11-10 at 9 32 18 am" src="https://cloud.githubusercontent.com/assets/2096628/20187323/aac0bb08-a728-11e6-8490-a7c86b683048.png">

**Comment by saliormoon** [comment]
Ok, thanks very much, @blink1073 .

**Comment by aoxiangcao** [comment]
hi,i'm doing the same thing just like you @saliormoon , i want to use jupyter in my web application, just like kaggle did. https://www.kaggle.com/artsiomrylau/d/ludobenistant/hr-analytics/human-resources-analytics-notebook  so ,what i should do? thanks a lot @blink1073

**Comment by blink1073** [comment]
Hi @aoxiangcao, this [example](https://github.com/jupyterlab/jupyterlab/tree/master/examples/notebook) is still the best place to start.
